tools/spell/README.md

```diff
@@ -13,9 +13,10 @@
 
 # How to create a lexicon.txt for a new library.
 
-1. Ensure there does not exist a file called "lexicon.txt" in your library's directory. Run the following command to create a lexicon.txt for your library:  
+1. Ensure there does not exist a file called "lexicon.txt" in your library's directory. Run the following command to create a lexicon.txt for your library:
    ```shell
-   find-unknown-comment-words -d <CSDK_ROOT>/libraries/<LIBRARY_TYPE>/<MY_LIBRARY_NAME> > <CSDK_ROOT>/libraries/<LIBRARY_TYPE>/<MY_LIBRARY_NAME>/lexicon.txt
```

Comment:
These instructions were incorrect. This is because when the lexicon.txt using the redirection command, the script will intake in immediately during operation, therefore not properly creating a new one.